draft-irtf-cfrg-vdaf.md

```diff
@@ -1582,17 +1582,18 @@ struct {
   MessageType type;
   select (Message.type) {
     case initialize:
-      opaque prep_share<0..2^32-1>;
+      opaque prep_share<1..2^32-1>;
     case continue:
       opaque prep_msg<0..2^32-1>;
-      opaque prep_share<0..2^32-1>;
+      opaque prep_share<1..2^32-1>;
     case finish:
       opaque prep_msg<0..2^32-1>;
   };
 } Message;
 ~~~
 
-These messages are used to transition between the states described in
+Note that the shortest encodable ping-pong message is `5` bytes long. These
```

Comment:
```suggestion
Note that the shortest valid ping-pong message is `5` bytes long. These
```

Why does the reader need to know how long the shortest valid message is?  If it matters, you could get it down to 2 bytes using the [variable-length headers](https://www.rfc-editor.org/rfc/rfc9420.html#name-variable-size-vector-length) introduced in MLS.